flow/Makefile

```diff
@@ -182,6 +182,9 @@ ifdef BLOCKS
   export GDS_FILES += $(BLOCK_GDS)
 endif
 
+export RTLMP_RPT_DIR ?= $(OBJECTS_DIR)/rtlmp
+export RTLMP_RPT_FILE ?= partition.txt
```

Comment:
RTLMP_RPT_DIR and RTLMP_RPT_FILE were set in designs config_hier.mk. Moving this to Makefile, means we can remove from the individual config files